app/views/membership_applications/_search_form.html.haml

```diff
@@ -1,6 +1,17 @@
 = search_form_for @search_params, method: :get, url: membership_applications_path,
                                   class: 'form-horizontal' do |f|
   .form-group
+    .col-sm-2
+      = f.label :user_membership_number_in,
+                "#{t('membership_applications.index.membership_number')}",
+                class: 'control-label',
+                style: 'text-align: left; font-size: 12px;'
+      = f.collection_select :user_membership_number_in,
+                      User.distinct.order(:membership_number).pluck(:membership_number).reject(&:blank?),
```

Comment:
This AR relation should be in the model.   Also, please look at using `map` rather than `pluck` as I think that the former could be faster than the latter.  (it may depend on the situation, but I had benchmarked a specific query using both and `map` was significantly faster the `pluck` - this was counter-intuitive as you'd think that pluck - pulling one column instead of a whole record - would be faster ...)

Reply:
very interesting about the map vs. pluck benchmark!

Reply:
I agree. And I think that that comparison (together with moving the AR relation to the model) is best left as part of the https://www.pivotaltracker.com/story/show/151966462 which has the focus on refactoring the search forms. I put that as a task in the story. 

My take on putting in PR's is that I try to focus on the story itself, trying to implement them while keeping the existing code as unchanged as possible and following the existing code practices - leaving possible or desirable refactorings to dedicated stories, although that might sometimes mean more or double work. I hope that's no problem.

Reply:
Yes, point well taken, especially since there are other occurrences of this in the same file.  That is the best way to handle it.

(and I'm pretty sure that I was the first sinner here :) )